cli/cmd/environment.go

```diff
@@ -31,6 +34,22 @@ Or just save it to a file
 		username := os.Getenv("FASIT_USERNAME")
 		password := os.Getenv("FASIT_PASSWORD")
 
+		// Fall back to the ident of the logged in user
+		if username == "" {
+			username = os.Getenv("USER")
+		}
+
+		if password == "" {
+			fmt.Fprintf(os.Stderr, "Enter ident password: ")
```

Comment:
I would remove `ident`, as you sometimes can use service users and so on.

Reply:
I agree, its kinda redundant anyway

Reply:
Maybe we could print the value of `username` instead? `Enter password for <username>`?

Reply:
Printing the username here, removes the need for entering the user earlier. I just want a way to know which user is used.

Reply:
Makes sense, I'll fix it when I get back home.